docs/en/ingest-management/migrate-beats-to-agent.asciidoc

```diff
@@ -0,0 +1,513 @@
+previous policy. To do this:
++
+--
+. In {kib}, go to *Stack Management > Index Lifecycle Policies* and search for a
+{beats} policy, for example, *filebeat*. Under *Linked indices*, notice you can
+view indices linked to the policy. Click the policy name to see the settings.
+
+. Click the *logs* policy and, if necessary, edit the settings to match the old
+policy.
+
+. Under *Index Lifecycle Policies*, search for another {beats} policies, for
+example, *metricbeat*.
+
+. Click the *metrics* policy and edit the settings to match the old policy.
+--
++
+Optionally delete the {beats} index lifecycle policies when they are no longer
+used by an index.
+
+* *Keep the {beats} policy and apply it to the index templates created for data
+streams.* To preserve an existing policy, modify it, as needed, and apply it to
+all the index templates created for data streams:
++
+--
+. Under *Index Lifecycle Policies*, find the {beats} policy, for example,
+*filebeat*.
+
+. In the *Actions* column, click the *Add policy to index template* icon.
+
+. Under *Index template*, choose a data stream index template, then add the
+policy.
+
+. Repeat this procedure, as required, to apply the policy to other data stream
+index templates.
+--
+
+//QUESTION: The UI gives you the option of setting the rollover alias. Should
+//we say anything about that here, or just assume they'll leave it blank?
+
+.What if you're not using ILM with {beats}?
+****
+You can begin to use ILM now with {agent}. Under *Index lifecycle policies*,
+click the policy name and edit the settings to meet your requirements.
+****
+
+[discrete]
+== Remove {beats} from your host
+
+Any installed {beats} shippers will continue to work until you remove them. This
+allows you to roll out the migration in stages. You will continue to see
+duplicated data until you remove {beats}.
+
+When you're satisfied with your {agent} deployment, remove {beats} from your
+hosts. All the data collected before installing the {agent} will still be
+available in {es} until you delete the data or it's removed according to the
+data retention policies you've specified for ILM.
+
+To remove {beats} from your host:
+
+//REVIEWERS: I'm just making stuff up below to show you the types of info I think
+//we need to provide. We don't really document how to uninstall Beats anyhwere. I might
+//be stating the obvious here, but I think we have to tell users how to clean
+//up their system. Let me know what's right/wrong/missing.
+
+. Stop the service by using the correct command for your system.
+
+. (Optional) Back up your {beats} configuration files in case you need to refer
+to to them in the future.
+
+. Delete the {beats} installation directory. If necessary, kill any orphan
+processes that are running after you stopped the service.
+
+. If you added firewall rules to allow {beats} to communicate on your network,
+remove them.
+
+. After you've removed all {beats}, revoke any API keys or remove privileges for
```

Comment:
Yes!!   I think these should be in the beats sections. lmk what you think?

Reply:
agreed. See my comment above.